src/screens/CreatePost/elements/CreatePollContainer/style/CreatePollContainerBaseStyle.js

```diff
@@ -63,15 +63,15 @@ const CreatePollContainerBaseStyle = StyleSheet.create({
   },
 
   polldurationbutton: {
-    backgroundColor: COLORS.white,
-    color: COLORS.white,
+    backgroundColor: colors.white,
+    color: colors.white,
     paddingVertical: 4,
     borderRadius: 4
   },
 
   polldurationbuttontext: {
-    color: COLORS.white,
-    backgroundColor: COLORS.holyTosca,
+    color: colors.white,
+    backgroundColor: colors.holytosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a logical inconsistency in your code. You've set the `backgroundColor` and `color` of `polldurationbutton` to `colors.white`, which means the text color and the button's background color will be the same, making the text invisible. Please revise this.

```diff
-    polldurationbutton: {
-        backgroundColor: colors.white,
-        color: colors.white,
-        paddingVertical: 4,
-        borderRadius: 4
-    },
+    polldurationbutton: {
+        backgroundColor: colors.white,
+        color: colors.black, // or any other contrasting color
+        paddingVertical: 4,
+        borderRadius: 4
+    },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->